### ISSUE TITLE
Attempted import error: 'rc-textarea' does not contain a default export (imported as 'RcTextArea').

### ISSUE THREAD

**Issue by YouDonggeun** [ant-design/ant-design#25285]
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[![Edit on CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/antd-reproduction-template-kr5uz?fontsize=14&hidenavigation=1&theme=dark)

### Steps to reproduce
install antd and exec any example 

### What is expected?
"no error"

### What is actually happening?
./node_modules/antd/es/input/TextArea.js
Attempted import error: 'rc-textarea' does not contain a default export (imported as 'RcTextArea').

| Environment | Info |
|---|---|
| antd | 4.4.0 |
| React | ^16.13.1 |
| System | MAC |
| Browser | CHROME 83 |

---

```bash
./node_modules/antd/es/input/TextArea.js:26
import RcTextArea from 'rc-textarea';
```
some dependencies is might be dropped

**Comment by karelsoupaEMZ** [comment]
I'm getting the same error. 4.4.0 unusable

**Comment by ycjcl868** [comment]
+1

**Comment by duskmoon314** [comment]
I'm getting the same error. 😢

**Comment by ycjcl868** [comment]
https://github.com/react-component/textarea/commit/92fd5e99339cec6da43fdff6c474928860daf4df

**Comment by JavaScripti** [comment]
'm getting the same error. 4.4.0 unusable

**Comment by eparizzi** [comment]
Same issue here when using Antd 4.4.0 in a Next project:
https://codesandbox.io/s/antd-440-with-next-issue-i0qb2

**Comment by chaitan94** [comment]
> I tried to use `rc-textarea: 0.2.2`, it's fine.

Are you sure? Rolling back `antd` to `4.3.5` or `rc-textarea` to `0.2.2` or even doing both doesn't work for me.
```bash
$ yarn add antd@4.3.5 rc-textarea@0.2.0
```

**Comment by ycjcl868** [comment]
> > I tried to use `rc-textarea: 0.2.2`, it's fine.
> 
> Are you sure? Rolling back `antd` to `4.3.5` or `rc-textarea` to `0.2.2` or even doing both doesn't work for me.
> 
> ```shell
> $ yarn add antd@4.3.5 rc-textarea@0.2.0
> ```

you need `resolutions` to install `rc-textarea` because antd depends on `rc-textarea`

```
 "resolutions": {
    "@umijs/plugin-qiankun": "2.3.0-7"
  },
```

**Comment by Kermit-Xuan** [comment]
Fixed in `rc-textarea@0.2.6`
